### PR TITLE
MCLOUD-5563: Add information about warm-up concurrency into log

### PR DESCRIPTION
**Pull request by NadiyaS**
### Description

Add information in cloud.log about concurrency which is set in WARM_UP_CONCURRENCY configuration 

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-5563

### Manual testing scenarios

- Set WARM_UP_CONCURRENCY configuration
- Cloud.log should have information about this used concurrency

### Release notes

Updated the cloud.log to include the concurrent request limit for sending requests during cache warmup. This value is configured in  in the [WARM_UP_CONCURRENCY post-deploy variable](https://devdocs.magento.com/cloud/env/variables-post-deploy.html#warm_up_concurrency).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] Pull request introduces user-facing changes and includes meaningful updates for any required release notes and documentation changes
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by oshmyheliuk — DISMISSED** [review]

**Review by oshmyheliuk — DISMISSED** [review]

**Review by BarnyShergold — DISMISSED** [review]

File: src/Test/Unit/Step/PostDeploy/WarmUpTest.php [hunk 1]
```diff
@@ -185,6 +185,14 @@ public function testExecuteWithConcurrency()
         ];
         $concurrency = 2;
 
+        $this->loggerMock->expects($this->any())
+            ->method('info')
+            ->withConsecutive(
+                ['Starting page warming up'],
+                ['Warming up concurrency set to ' . $concurrency . ' which taken from '
```
Comment (meker12) [inline src/Test/Unit/Step/PostDeploy/WarmUpTest.php]:
```suggestion
                ['Warmup concurrency set to ' . $concurrency . ' as specified by the  '
```

File: src/Test/Unit/Step/PostDeploy/WarmUpTest.php [hunk 2]
```diff
@@ -185,6 +185,14 @@ public function testExecuteWithConcurrency()
         ];
         $concurrency = 2;
 
+        $this->loggerMock->expects($this->any())
+            ->method('info')
+            ->withConsecutive(
+                ['Starting page warming up'],
```
Comment (meker12) [inline src/Test/Unit/Step/PostDeploy/WarmUpTest.php]:
```suggestion
                ['Starting page warmup'],
```

File: src/Step/PostDeploy/WarmUp.php [hunk 3]
```diff
@@ -70,6 +70,18 @@ public function execute()
             $this->logger->info('Starting page warming up');
             $config = [];
 
+            $concurrency = $this->postDeploy->get(PostDeployInterface::VAR_WARM_UP_CONCURRENCY);
+            if ($concurrency) {
+                $config['concurrency'] = $concurrency;
+                $this->logger->info(
+                    sprintf(
+                        'Warming up concurrency set to %s which taken from %s configuration',
```
Comment (meker12) [inline src/Step/PostDeploy/WarmUp.php]:
```suggestion
                        'Warmup concurrency set to %s as specified by the %s configuration',
```

**Review by meker12 — DISMISSED** [review]

**Review by oshmyheliuk — DISMISSED** [review]

**Review by BarnyShergold — APPROVED** [review]

**Review by oshmyheliuk — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**